<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialSettingsWidget</class>
 <widget class="QWidget" name="SerialSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>124</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>124</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>69</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Active</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>532</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="1" column="4">
     <widget class="QComboBox" name="stopBits">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QComboBox" name="dataBits">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Baud</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Stop bits</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="baud">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>230400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>250000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>460800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>500000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>921600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1000000</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QComboBox" name="parity">
      <item>
       <property name="text">
        <string>none</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>odd</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>even</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="serialPort">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>/dev/ttyS4</string>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Serial port</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Data bits</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Parity</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkListen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>93</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Listen to</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="listenPort">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>/dev/ttyUSB0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
